frontend/src/features/leases/detail/LeasePages/surplus/Surplus.test.tsx

```diff
@@ -0,0 +1,96 @@
+import { Formik } from 'formik';
+import { createMemoryHistory } from 'history';
+import { defaultFormLease, IFormLease, IProperty } from 'interfaces';
+import { noop } from 'lodash';
+import { mockOrganization, mockProperties, mockUser } from 'mocks/filterDataMock';
+import { prettyFormatDate } from 'utils';
+import { render, RenderOptions, RenderResult } from 'utils/test-utils';
+
+import Surplus from './Surplus';
+
+const history = createMemoryHistory();
+
+describe('Lease Surplus Declaration', () => {
+  const setup = (renderOptions: RenderOptions & { lease?: IFormLease } = {}): RenderResult => {
+    // render component under test
+    const result = render(
+      <Formik onSubmit={noop} initialValues={renderOptions.lease ?? defaultFormLease}>
+        <Surplus />
+      </Formik>,
+      {
+        ...renderOptions,
+        history,
+      },
+    );
+    return result;
+  };
+
+  it('renders as expected', () => {
+    const result = setup({
+      lease: { ...defaultFormLease, persons: [mockUser], organizations: [mockOrganization] },
+    });
+    expect(result.asFragment()).toMatchSnapshot();
+  });
+  it('Snapshot matches', () => {
+    const result = setup({
+      lease: { ...defaultFormLease, persons: [mockUser], organizations: [mockOrganization] },
+    });
+    expect(result.asFragment()).toMatchSnapshot();
+  });
```

Comment:
This test seems to be a duplicate of the one above - right?

Reply:
oph yeah, totally. I think I was tinkering with the asFragment method and forgot to remove it